package.json

```diff
@@ -43,68 +45,60 @@
     "@sentry/vue": "^7.49.0",
     "array-move": "^4.0.0",
     "axios": "^0.27.2",
-    "core-js": "^3.30.1",
     "http-status-codes": "^2.2.0",
     "launchdarkly-js-client-sdk": "^2.24.2",
     "lodash": "^4.17.21",
     "lodash.omit": "^4.5.0",
-    "pdfjs-dist": "3.4.120",
+    "pdfjs-dist": "3.9.179",
```

Comment:
This has been an issue for a while now.

The library works, however, I was having issues with unit tests.:
  - When I disable threads in `vite.config.js`, some tests were failing although they pass when I run them individually.
  - When I had threads enabled, I was receiving a canvas error that doesn't seem to have a fix.
![canvas errror](https://github.com/bcgov/business-create-ui/assets/122301442/ff1caf26-94ba-4b16-b82e-23ff16e80fa8)
People with same issue were running into this with pdfjs library:  https://github.com/vitest-dev/vitest/issues/740#issuecomment-1571588819

I've tried everything until I stumbled upon this version of the pdfjs library that was released 3 days ago. It fixed my issue 😁 

Tests now run super fast since we have threads on and everything looks good!

Reply:
Sometimes, it's better to be lucky than good.

(You're both 😁 )

Reply:
Haha I remember when me and Severin tried the latest library it wouldn't even work.. they must have fixed it since then..

Reply:
Great work Karim! Had issues with this during Pinia upgrade, it was frustrating